web/source/dom/uiTouchHandlerBase.ts

```diff
@@ -205,6 +237,10 @@ namespace com.keyman.dom {
         return;
       }
 
+      // Establish scroll tracking.
+      let scrollMode = (this.currentTarget.clientWidth < this.currentTarget.scrollWidth);
```

Comment:
I suggest changing this name to `shouldScroll`; then the line after this reads as "if shouldScroll then make a touch state"